tests/stdlib/tstrmisc.nim

```diff
@@ -0,0 +1,22 @@
+import strmisc
+
+
+func main() =
+  doAssert parseFloatThousandSep("0") == 0.0
+  doAssert parseFloatThousandSep("-0") == -0.0
+  doAssert parseFloatThousandSep("0.0") == 0.0
```

Comment:
are these supposed to be valid? currently they are:
```
nim> echo parseFloatThousandSep("1,")
1.0
nim> echo parseFloatThousandSep("1,1")
11.0
```

Reply:
I think so. :slightly_smiling_face:

Reply:
why though? seems too lax (ie, won't catch bugs).

* note that `locale.atof` accepts it but also accepts arbitrary gargage like `locale.atof('12,3,,4')` so isn't a comparison point
* you previously compared against python3 `float()` but also not a comparison point since `float('1,000')` isn't accepted
* is there another reference to justify these ?

Reply:
I agree that `1,` or `1,1` by default should not be accepted (both can only work if the decimal "point" is supposed to be `,` and the first even only in a more relaxed way than I'm personally happy with).